assets/js/atomic/blocks/product/add-to-cart/product-types/variable/variation-attributes/style.scss

```diff
@@ -0,0 +1,23 @@
+.wc-block-components-product-add-to-cart-attribute-picker {
+	margin: 0;
+	flex-basis: 100%;
+
+	label {
+		display: block;
+		@include font-size(regular);
+	}
+
+	.wc-block-components-product-add-to-cart-attribute-picker__select {
+		margin: 0 0 em($gap-small) 0;
+
+		select {
+			@include font-size(regular);
+			font-family: inherit;
+			min-width: 60%;
+			min-height: 1.75em;
+			border: 1px solid $input-border-gray;
+			border-radius: 4px;
+			margin: 0;
```

Comment:
I'm still not sure about styling the native select. :thinking:  Setting a custom border and border-radius makes the select look completely different from how it would look natively. And it makes it stand out compared to the select from the All Products and Reviews blocks.

| | Firefox | Chrome | Web (Webkit engine) |
| --- | --- | --- | --- |
| Product block | ![imatge](https://user-images.githubusercontent.com/3616980/85033648-7b8da780-b181-11ea-96b0-60ac29f87657.png) | ![imatge](https://user-images.githubusercontent.com/3616980/85033703-88aa9680-b181-11ea-8ec8-a08bfa268f9e.png) | ![imatge](https://user-images.githubusercontent.com/3616980/85033791-a24bde00-b181-11ea-8eb0-922cb0c5c22d.png) |
| All Products block | ![imatge](https://user-images.githubusercontent.com/3616980/85037668-f658c180-b185-11ea-8940-a165bfc56d5b.png) | ![imatge](https://user-images.githubusercontent.com/3616980/85037686-fe186600-b185-11ea-857e-0ca6138103fb.png) | ![imatge](https://user-images.githubusercontent.com/3616980/85037718-0670a100-b186-11ea-8ad2-75ee07fdb86b.png) |

Also, it was not clear to me why we need to set a font size here? Can't we just inherit the one from the theme? Looking at the WC Core product page, it looks like the only style properties the select has are `max-width` and `vertical-align`. Could we follow that approach here too and only write CSS for the layout, but not the specific styles?